tuna/utils/logger.py

```diff
@@ -30,28 +30,48 @@
 from tuna.miopen.utils.metadata import TUNA_LOG_DIR
 
 
-def setup_logger(logger_name='Tuna',
-                 add_streamhandler=True,
-                 add_filehandler=False):
+def setup_logger(logger_name: str = 'Tuna',
+                 add_streamhandler: bool = True,
+                 add_filehandler: bool = False) -> logging.Logger:
   """std setup for tuna logger"""
-  log_level = os.environ.get('TUNA_LOGLEVEL', 'INFO').upper()
+  log_level: str = os.environ.get('TUNA_LOGLEVEL', 'INFO').upper()
   logging.basicConfig(level=log_level)
-  logger = logging.getLogger(logger_name)
-  log_file = os.path.join(TUNA_LOG_DIR, logger_name + ".log")
-  formatter = logging.Formatter(
+  logger: logging.Logger = logging.getLogger(logger_name)
+  log_file: str = os.path.join(TUNA_LOG_DIR, logger_name + ".log")
+  formatter: logging.Formatter = logging.Formatter(
       '%(asctime)s - %(name)s - %(levelname)s -  [%(filename)s:%(lineno)d] - %(message)s'
   )
   logger.propagate = False
+
   if add_filehandler:
-    file_handler = logging.FileHandler(log_file, mode='a')
+    file_handler: logging.FileHandler = logging.FileHandler(log_file, mode='a')
     file_handler.setFormatter(formatter)
     file_handler.setLevel(log_level.upper() if log_level else logging.INFO)
     logger.addHandler(file_handler)
   if add_streamhandler:
-    stream_handler = logging.StreamHandler()
+    stream_handler: logging.StreamHandler = logging.StreamHandler()
     stream_handler.setFormatter(formatter)
     stream_handler.setLevel(logging.INFO)
     logger.addHandler(stream_handler)
 
   logger.setLevel(log_level.upper() if log_level else logging.DEBUG)
   return logger
+
+
+def set_usr_logger(logger_name: str) -> logging.Logger:
+  """Build logger with user defined name"""
```

Comment:
So both `set_usr_logger` and `setup_logger` take in a logger_name and create a logger with that name. Maybe update the docstring to better reflect exactly how set_usr_logger differs from setup_logger

Reply:
log_level: str = os.environ.get('TUNA_WOKER_INTERFACE_LOGLEVEL', "INFO")
Have added this already. this differenctiates the logger type